spec/models/miq_report_spec.rb

```diff
@@ -22,16 +11,7 @@ def invoke_ae
 
     @resource = base_report == "Host" ? FactoryGirl.create(:host) : FactoryGirl.create(:vm_vmware)
     FactoryGirl.create(custom_attribute_field, :resource => @resource, :name => @key, :value => @value)
-
-    if custom_attribute_field == :ems_custom_attribute
-      custom_get_method = "ems_custom_get"
-    else
-      custom_get_method = "custom_get"
-    end
-
-    method = "$evm.root['#{ae_result_key}'] = $evm.root['#{base_report_downcase}'].#{custom_get_method}('#{@key}')"
-    @ae_method.update_attributes(:data => method)
-    @ae_object = invoke_ae.root(ae_result_key)
+    @resource.miq_custom_set(@key, @value)
```

Comment:
@imtayadeway I was wondering why it is working properly also for EMS custom attributes when you are calling only `miq_custom_set(@key, @value)` and I figure out that this method is not needed to call because custom attributes and it Miq or EMS are created on line 13. It is thanks to factory girl https://github.com/ManageIQ/manageiq/blob/master/spec/factories/custom_attribute.rb#L9  (field source   'EVM'  are related to Miq custom attr. and field  source   'VC' is for Miq) 

so In the end it seems this line can be removed